### PR TITLE
Mounts sidekiq web monitor

### PR DESCRIPTION
**Pull request by benlovell**
Uses the same username and password as the admin control panel.

### CODE REVIEW

**Comment by asmega** [comment]
:+1:

**Comment by esbel** [comment]
:+1:

File: config/routes.rb [hunk 1]
```diff
@@ -40,4 +42,10 @@
     end
     resources :principals, only: [:index, :show]
   end
+
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    Authentication.authenticate(username, password)
+  end if Rails.env.production?
```
Comment (jongilbraith) [inline config/routes.rb]: As discussed, stylistically my preference would be to wrap the whole block in the if statement.

**Comment by jongilbraith** [comment]
Other than that small comment :+1:

**Pull request merged**